packages/azure-services/src/azure-queue/queue.ts

```diff
@@ -71,10 +74,14 @@ export class Queue {
     }
 
     public async createMessage(queue: string, message: unknown): Promise<boolean> {
-        const queueURL = await this.getQueueURL(queue);
-        await this.ensureQueueExists(queueURL);
-
-        return this.createQueueMessage(queueURL, message);
+        return this.retryHelper.executeWithRetries(
+            () => this.tryCreateMessage(queue, message),
+            async (error: Error) => {
+                return;
+            },
+            this.maxAttempts,
+            this.millisBetweenRetries,
```

Comment:
you don't have to expose millisBetweenRetries via contructor. Instead to exponential delay like we do in NotificationQueueMessageSender & remove the retry logic from there.

Reply:
Our retry helper does not support exponential delay right now.

Reply:
with this code, then you end up retrying like 6 times for notificationRequestSender

Reply:
I'm not following, the retry helper will only try up to the maxAttempts which is 3 here.

Reply:
check this code - https://github.com/microsoft/accessibility-insights-service/blob/1ff2560a03e7cbdca317375b8299e4b9e50f4ea3/packages/web-api-scan-runner/src/tasks/notification-queue-message-sender.ts#L58

Reply:
Why not use the retry helper that we already have?